src/index.html

```diff
@@ -5,8 +5,316 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="img"
+        >
+      </a>
+
+      <nav class="nav" >
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+            href="#"
+            data-qa="nav-hover"
+            >
```

Comment:
Pay attention on indents. Fix all similar cases
```suggestion
            <a class="nav__link"
              href="#"
              data-qa="nav-hover"
            >
```